docs/ManualDeploymentGuideScalarDLOnAWS.md

```diff
@@ -222,12 +246,21 @@ When you need to remove the resources that you have created, remove the resource
 You can uninstall Scalar DL installation with the following Helm commands:
 
    ```console
-    # Uninstall loaded schema with a release name 'load-schema'
-    helm delete load-schema
+   # Uninstall loaded schema with a release name 'load-schema'
+   helm delete load-schema
 
-    # Uninstall Scalar DL with a release name 'my-release-scalardl'
-    helm delete my-release-scalardl
+   # Uninstall Scalar DL with a release name 'my-release-scalardl'
+   helm delete my-release-scalardl
    ```
+You can uninstall Scalar Auditor with the following Helm commands:
+
+    ```console
```

Comment:
This code block is not properly formatted for the same reason as https://github.com/scalar-labs/scalar-kubernetes/pull/150/files#r743349918

Reply:
Thank you for your suggestions. 
Document formatted.